webapp/components/survey/CategoryDetails/store/actions/item/useUpdateItemProp.js

```diff
@@ -0,0 +1,42 @@
+import { useCallback } from 'react'
+import { useDispatch } from 'react-redux'
+import axios from 'axios'
+
+import { AppSavingActions } from '@webapp/store/app'
+import { SurveyActions, useSurveyId } from '@webapp/store/survey'
+
+import { debounceAction } from '@webapp/utils/reduxUtils'
+import { State } from '../../state'
+
+const _putPropAction = ({ surveyId, categoryUuid, itemUuid, key, value, setState, dispatch }) => async () => {
```

Comment:
dispatch is the argument of the returned async function. that's a redux thunk action

Reply:
right, I'll fix all of them